adr/2022-08-decouple-from-ietf.md

```diff
@@ -0,0 +1,137 @@
+# Decoupling from IETF
+
+* Status: accepted
+* Deciders: @jdesrosiers @relequestual @awwright
+* Date: 2022-08-17
+
+Related Issues:
+* https://github.com/orgs/json-schema-org/discussions/69 - This issue is about
+  dropping the "draft" prefix from our releases. This ADR doesn't cover that,
+  but much of the discussion about whether or not to decouple from IETF is in
+  that discussion.
+* This topic has been discussed in many other places as well that are difficult
+  to link to here including Slack, Twitter, OCWMs, and conference discussions.
+
+## Context and Problem Statement
+
+Currently JSON Schema loosely follows the IETF Internet-Draft (I-D) process for
+spec development and releases but isn't associated with any IETF working group.
+JSON Schema is an individual draft. That means it isn't on a standards track
+with IETF and IETF is not involved nor supports the spec in any way other than
+hosting the canonical version of our I-Ds. Our perceived involvement with IETF
+causes confusion and misunderstanding within our community in the cases were our
+practices and the realities of our situation deviate from a typical IETF I-D
+lifecycle. The thing that makes our situation different than a typical I-D is
+that our "drafts" are intended for use in production.
+
```

Comment:
Can we add a note here that we will continue to pursue registering the JSON Schema media types through an RFC published by the the IETF HTTPAPIs working group?  I think including that up front in the "Context" will reassure folks that a.) we're not totally abandoning internet standards, and b.) we actually do have some experience with working groups and are not dismissing the approach blindly.

Reply:
Good catch. It's important that we include our decision about media type registration. However, I think it should be in the "Decision Outcome" section not the problem statement.